cli/deployment/config_test.go

```diff
@@ -86,14 +87,34 @@ func TestFindAllDependencies(t *testing.T) {
 	}
 }
 
-func TestYAML(t *testing.T) {
+func TestYAMLStripCustomFields(t *testing.T) {
 	data, err := Config{
 		data: GetTestData("config", "custom-fields.yaml", t),
-	}.YAML()
+	}.YAML(map[string]map[string]interface{}{})
 	if err != nil {
 		t.Error(err)
 	}
 	if strings.Contains(string(data), "project:") {
 		t.Errorf("Should not contain, project, name or descriptions")
 	}
 }
+
+func TestYamlReplaceOutRefs(t *testing.T) {
+	data := GetTestData("cross-ref", "main-manifest.yaml", t)
+	output, err := parseOutputs(data)
+	if err != nil {
+		t.Errorf("failed to parse outputs: %v", err)
+	}
+	config := NewConfig(GetTestData("cross-ref", "dependent-with-refs.yaml", t), "/user/home/test.yaml")
+	outputs := map[string]map[string]interface{}{}
+	outputs["prj1.name1"] = output
+	actual, err := config.YAML(outputs)
+	if err != nil {
+		t.Errorf("failed to export config YAML: %v", err)
+	}
+	ioutil.WriteFile("/1-prj/projects/google/cloud-foundation-toolkit/cli/testdata/cross-ref/test", actual, 0755)
```

Comment:
This file path probably isn't going to exist for most people who check out this repo. Can you make this a relative path?